teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @specialization = specialization
+    @parent_permission = parent_permission
```

Comment:
- [ ] Please remove this line from your file because `parent_permission` should only pass to the person class, by adding this line, you are overriding the parent_permission of the parent class, which is not a good idea. Because the person's personal data should be handled from the parent class not from the `extended` class.